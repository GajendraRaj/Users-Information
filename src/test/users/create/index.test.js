import React from "react";
import { shallow } from "enzyme";
import CreateList from "../../../components/users/create";

describe("CreateList component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CreateList />);
  });

  it("should render component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render Create user button to add new record", () => {
    expect(wrapper.find("button").at(0).text()).toEqual("Create New User");
  });

  it("should render model popup on 'Create New User' button click", () => {
    const createButton = wrapper.find("button").at(0);
    createButton.simulate("click");
    expect(wrapper.find("th").at(0).text()).toEqual("Create New User");
  });
});
