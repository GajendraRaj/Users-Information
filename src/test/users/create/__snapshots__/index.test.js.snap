// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateList component should render component correctly 1`] = `ShallowWrapper {}`;
